docs/machine_specific_instructions/linux.rst

```diff
@@ -84,5 +84,5 @@ Then, when you want to build or run MPAS-Ocean, source a file containing:
     export PNETCDF=${PREFIX}
     export PIO=${PREFIX}
     # change to one of the other cores as needed
-    export CORE=ocean
+    export
```

Comment:
This looks like a mistake?  Presumably this whole line should be deleted.

Reply:
```suggestion
```